nucypher-core-wasm/src/lib.rs

```diff
@@ -244,6 +287,10 @@ impl Address {
 #[derive(PartialEq, Debug, derive_more::From, derive_more::AsRef)]
 pub struct MessageKit(nucypher_core::MessageKit);
 
+generate_equals!(MessageKit);
+generate_from_bytes!(MessageKit);
+generate_to_bytes!(MessageKit);
```

Comment:
This is great - can is be abstracted farther or does that cross some kind of line? 😆

Reply:
We could put these into `generate_common_methods!` but I didn't find it as useful.